RazorGenerator.Tooling/source.extension.vsixmanifest

```diff
@@ -1,24 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011">
   <Metadata>
-    <Identity Id="2FD58EB3-29A2-40A3-BE17-C024CA9A17A2" Version="1.12.0" Language="en-US" Publisher="RazorGenerator contributors" />
-    <DisplayName>Razor Generator</DisplayName>
+    <Identity Id="5CEC18E0-07C4-4663-9228-A8D88ED57EA0" Version="1.13.0" Language="en-US" Publisher="Guillaume Zahra" />
+    <DisplayName>Razor Generator 2022</DisplayName>
     <Description>Generates source code from Razor files (.cshtml/.vbhtml files), allowing them to be compiled into your assemblies. Supports MVC, Web Pages and standalone templates.</Description>
     <MoreInfo>https://github.com/RazorGenerator/RazorGenerator</MoreInfo>
     <License>License.txt</License>
     <GettingStartedGuide>https://github.com/RazorGenerator/RazorGenerator/wiki/Quick-Start-Guide</GettingStartedGuide>
   </Metadata>
   <Installation>
-    <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Community" />
-    <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Pro" />
-    <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Enterprise" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,)">
```

Comment:
Why not have it as `[11.0,)` so we don't block older VS?

Reply:
Only because i didn't had the time to test the evolution.

Actually, i removed all the COM reference needed on previous Razor Generator versions, and migrated to the NuGet package model equivalent recommended by Microsoft for VS2022 extensions support, and maybe some other stuff.

I didn't searched nor tested if this new model is backward compatible and if it's working out of the box.
Maybe remove the soft limit i set, and testing on a previous version of Visual Studio "just work" but there is a lot of versions to test possibly.

You can imagine, i also done this because i needed this tool for a Business project (and because i love using the last versions of VS) so the main goal was to have something shippable for myself.

Otherwise, yes of course, if it works on older version, you are totally right about keeping the old **[11.0,)** argument in the manifest

Reply:
It very likely doesn't work well on 11.0, which is VS 2012, but I don't think anyone care. :) How about we set the minimum to 15.0 (VS 2017)? Even if we don't have time to test it, at least we're not blocking it entirely. Then if someone reports issue, we can always look into it at that point.

Reply:
Should be a good idea i think !